src/openzeppelin/access/accesscontrol/accesscontrol.cairo

```diff
@@ -84,14 +84,14 @@ mod AccessControl {
 
 
     #[view]
-    fn supports_interface(interface_id: u32) -> bool {
-        ERC165::supports_interface(interface_id)
+    fn supports_interface(interface_id: felt252) -> bool {
+        SRC5::supports_interface(interface_id)
     }
 
     #[view]
-    fn supportsInterface(interface_id: u32) -> bool {
+    fn supportsInterface(interface_id: felt252) -> bool {
         // TODO: Change to camel when available
```

Comment:
Does when available imply that we should add camelCase to SRC5 module?

Reply:
Mmm good question. The answer's no, right? This was with the backwards compatibility idea in mind with erc165. I suppose we should just omit the camel `supportsInterface` for now and add it back if we include dual introspection (with erc165 depending on #640). What do you think?

Reply:
Regarding SRC5, exposing the camelCase interface is for backward compatibility, right? Since there are no previous SRC5 modules, we don't have existing contracts calling SRC5 as camelCase, so I think we don't need to expose it, and is enough with just snake_case. 

I think we should keep the camelCase interface only if we are maintaining the ERC165 backward compatibility, so I think merging  assuming breaking change (just SRC5) is ok, because the alternative would be to implement ERC165/SRC5 support, and that's the thing we are trying to avoid.

Reply:
Perfect, sounds good!